templates/login.html

```diff
@@ -2,11 +2,43 @@
 <html>
 
 <head>
-    <title>PieceMeal - Log In</title>
+  <title>PieceMeal - Log In</title>
+  <link rel="stylesheet" type="text/css" href='../static/style.css'>
+  <link rel="stylesheet" type="text/css" href='../static/login.css'>
 </head>
 
 <body>
+  <div class="alt-body1">
+    <div class="container">
+      <img class="logo" src="../static/logo.png" alt="PieceMeal" />
+      <div class="G-Login">
+        <input id="google-button" type="button" class="google-img" href="/google_auth" />
+        <img class="googlelog" />
+      </div>
+
+      <form class="credentials" action="/account" method="POST">
+        <p>or</p>
+        <p>Sign in via our site</p>
+        <input type="text" placeholder="Enter your email..." />
+        <input type="password" placeholder="Enter your password..." />
+        <button type="submit">Login</button>
+      </form>
```

Comment:
We don't have our own authentication system. We are only authenticating through Google (at least for now anyways).